src/sonic-config-engine/sonic_platform.py

```diff
@@ -21,7 +21,7 @@ def get_machine_info():
             tokens = line.split('=')
             if len(tokens) < 2:
                 continue
-            machine_vars[tokens[0]] = tokens[1].strip()
+            machine_vars[tokens[0]] = tokens[1].strip().strip("'")
```

Comment:
do not modify this file.

Reply:
OK, but we found this method return machine name with the single quote mark